tests/cases/fourslash/codeFixAddMissingProperties16.ts

```diff
@@ -0,0 +1,166 @@
+/// <reference path='fourslash.ts' />
+
+////type A = 'a' | 'b' | 'c' | 'd' | 'e';

```

Comment:
are you testing scale? If not, I don't think we need 5 or even 3 entries to that the codefix works; other tests already show that string literal types distribute across unions.

Reply:
No, I don't. I've removed other similar tests.